docs/conf.py

```diff
@@ -69,8 +69,16 @@
     "gitter_room": "pydata/pandas",
     "open_collective": "pandas",
     "show_powered_by": True,
-    "github_url": "https://github.com/pandas-dev/pandas",
-    "twitter_url": "https://twitter.com/pandas_dev"
+    "footer_links": [
+        {
+            'url': "https://github.com/pandas-dev/pandas",
+            'name': "GitHub"
```

Comment:
I was just thinking of having a single dict with the name as the key and the URL as the value, but I imagine this future-proofs it in case you want to add more parameters.